src/Pages/login/login.jsx

```diff
@@ -12,28 +13,30 @@ export default () => {
   const [userNameAlert, setUserNameAlert] = useState(null);
   const [passwordAlert, setPasswordAlert] = useState(null);
 
-  function checkUsername () {
+  const [isInputsValid, setInputValid] = useState(null);
```

Comment:
```suggestion
  const [isInputsValid, setIsInputsValid] = useState(false);
```
We should always try to keep the state in one type (in this case boolean) at any given moment to avoid unforeseen complications in code

Reply:
Fixed 8b7142b3bb568f0c1e361ef10bfeae717284050f